From f81bc02fcbdfe05f159c7173f75547c8214fee9c Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Wed, 26 Feb 2020 06:53:50 +0100
Subject: [PATCH 1/1] ata: sort ATA commands

Sort the ATA commands in include/libata.h by number.
Add a few more comments.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 include/libata.h | 84 ++++++++++++++++++++++++------------------------
 1 file changed, 42 insertions(+), 42 deletions(-)

diff --git a/include/libata.h b/include/libata.h
index 25296ac66d..b03b29960d 100644
--- a/include/libata.h
+++ b/include/libata.h
@@ -133,49 +133,49 @@ enum {
 	ATA_REG_IRQ		= ATA_REG_NSECT,
 
 	/* ATA device commands */
-	ATA_CMD_DEV_RESET	= 0x08, /* ATAPI device reset */
-	ATA_CMD_CHK_POWER	= 0xE5, /* check power mode */
-	ATA_CMD_STANDBY		= 0xE2, /* place in standby power mode */
-	ATA_CMD_IDLE		= 0xE3, /* place in idle power mode */
-	ATA_CMD_EDD		= 0x90,	/* execute device diagnostic */
-	ATA_CMD_FLUSH		= 0xE7,
-	ATA_CMD_FLUSH_EXT	= 0xEA,
-	ATA_CMD_ID_ATA		= 0xEC,
-	ATA_CMD_ID_ATAPI	= 0xA1,
-	ATA_CMD_READ		= 0xC8,
-	ATA_CMD_READ_EXT	= 0x25,
-	ATA_CMD_WRITE		= 0xCA,
-	ATA_CMD_WRITE_EXT	= 0x35,
-	ATA_CMD_WRITE_FUA_EXT	= 0x3D,
-	ATA_CMD_FPDMA_READ	= 0x60,
-	ATA_CMD_FPDMA_WRITE	= 0x61,
-	ATA_CMD_PIO_READ	= 0x20,
-	ATA_CMD_PIO_READ_EXT	= 0x24,
-	ATA_CMD_PIO_WRITE	= 0x30,
-	ATA_CMD_PIO_WRITE_EXT	= 0x34,
-	ATA_CMD_READ_MULTI	= 0xC4,
-	ATA_CMD_READ_MULTI_EXT	= 0x29,
-	ATA_CMD_WRITE_MULTI	= 0xC5,
-	ATA_CMD_WRITE_MULTI_EXT	= 0x39,
-	ATA_CMD_WRITE_MULTI_FUA_EXT = 0xCE,
-	ATA_CMD_SET_FEATURES	= 0xEF,
-	ATA_CMD_SET_MULTI	= 0xC6,
-	ATA_CMD_PACKET		= 0xA0,
-	ATA_CMD_VERIFY		= 0x40,
-	ATA_CMD_VERIFY_EXT	= 0x42,
-	ATA_CMD_STANDBYNOW1	= 0xE0,
-	ATA_CMD_IDLEIMMEDIATE	= 0xE1,
-	ATA_CMD_SLEEP		= 0xE6,
-	ATA_CMD_INIT_DEV_PARAMS	= 0x91,
-	ATA_CMD_READ_NATIVE_MAX	= 0xF8,
+	ATA_CMD_DEV_RESET	    = 0x08, /* ATAPI device reset */
+	ATA_CMD_PIO_READ	    = 0x20, /* Read sectors with retry */
+	ATA_CMD_PIO_READ_EXT	    = 0x24,
+	ATA_CMD_READ_EXT	    = 0x25,
 	ATA_CMD_READ_NATIVE_MAX_EXT = 0x27,
-	ATA_CMD_SET_MAX		= 0xF9,
-	ATA_CMD_SET_MAX_EXT	= 0x37,
-	ATA_CMD_READ_LOG_EXT	= 0x2f,
-	ATA_CMD_PMP_READ	= 0xE4,
-	ATA_CMD_PMP_WRITE	= 0xE8,
-	ATA_CMD_CONF_OVERLAY	= 0xB1,
-	ATA_CMD_SEC_FREEZE_LOCK	= 0xF5,
+	ATA_CMD_READ_MULTI_EXT	    = 0x29,
+	ATA_CMD_READ_LOG_EXT	    = 0x2f,
+	ATA_CMD_PIO_WRITE	    = 0x30, /* write sectors with retry */
+	ATA_CMD_PIO_WRITE_EXT	    = 0x34,
+	ATA_CMD_WRITE_EXT	    = 0x35,
+	ATA_CMD_SET_MAX_EXT	    = 0x37,
+	ATA_CMD_WRITE_MULTI_EXT	    = 0x39,
+	ATA_CMD_WRITE_FUA_EXT	    = 0x3D,
+	ATA_CMD_VERIFY		    = 0x40, /* read verify sectors with retry */
+	ATA_CMD_VERIFY_EXT	    = 0x42,
+	ATA_CMD_FPDMA_READ	    = 0x60,
+	ATA_CMD_FPDMA_WRITE	    = 0x61,
+	ATA_CMD_EDD		    = 0x90, /* execute device diagnostic */
+	ATA_CMD_INIT_DEV_PARAMS	    = 0x91, /* initialize device parameters */
+	ATA_CMD_PACKET		    = 0xA0, /* ATAPI packet */
+	ATA_CMD_ID_ATAPI	    = 0xA1, /* ATAPI identify device */
+	ATA_CMD_CONF_OVERLAY	    = 0xB1,
+	ATA_CMD_READ_MULTI	    = 0xC4, /* read multiple */
+	ATA_CMD_WRITE_MULTI	    = 0xC5, /* write multiple */
+	ATA_CMD_SET_MULTI	    = 0xC6, /* set multiple mode */
+	ATA_CMD_READ		    = 0xC8, /* read DMA with retry */
+	ATA_CMD_WRITE		    = 0xCA, /* write DMA with retry */
+	ATA_CMD_WRITE_MULTI_FUA_EXT = 0xCE,
+	ATA_CMD_STANDBYNOW1	    = 0xE0, /* standby immediate */
+	ATA_CMD_IDLEIMMEDIATE	    = 0xE1, /* idle immediate */
+	ATA_CMD_STANDBY		    = 0xE2, /* place in standby power mode */
+	ATA_CMD_IDLE		    = 0xE3, /* place in idle power mode */
+	ATA_CMD_PMP_READ	    = 0xE4, /* read buffer */
+	ATA_CMD_CHK_POWER	    = 0xE5, /* check power mode */
+	ATA_CMD_SLEEP		    = 0xE6, /* sleep */
+	ATA_CMD_FLUSH		    = 0xE7,
+	ATA_CMD_PMP_WRITE	    = 0xE8, /* write buffer */
+	ATA_CMD_FLUSH_EXT	    = 0xEA,
+	ATA_CMD_ID_ATA		    = 0xEC, /* identify device */
+	ATA_CMD_SET_FEATURES	    = 0xEF, /* set features */
+	ATA_CMD_SEC_FREEZE_LOCK	    = 0xF5, /* security freeze */
+	ATA_CMD_READ_NATIVE_MAX	    = 0xF8,
+	ATA_CMD_SET_MAX		    = 0xF9,
 
 	/* READ_LOG_EXT pages */
 	ATA_LOG_SATA_NCQ	= 0x10,
-- 
2.25.0

